src/patternfly/components/DataList/data-list.scss

```diff
@@ -251,6 +251,18 @@
     cursor: pointer;
     outline-offset: var(--pf-c-data-list__item--m-selectable--OutlineOffset);
 
+    &.pf-m-selected {
+      --pf-c-data-list__item--before--BackgroundColor: var(--pf-c-data-list__item--m-selected--before--BackgroundColor);
+
+      position: relative;
+      z-index: var(--pf-c-data-list__item--m-selected--ZIndex);
+
+      &:hover {
```

Comment:
I think this will need to also have :focus because when I use keyboard focus, I can still see the break in the border. 
<img width="1442" alt="image" src="https://user-images.githubusercontent.com/19825616/170113636-52dfd127-6859-4cce-8551-65903a872594.png">

Reply:
Hmm, this removes the bottom border on hover, which I think looks a bit off since the border is removed when you hover the top selected row, but it appears if you hover the bottom selected row.

Hovering the top selected row:
<img width="1846" alt="Screen Shot 2022-07-05 at 6 32 20 PM" src="https://user-images.githubusercontent.com/35148959/177433582-50c9ac08-8da7-49eb-80fe-93518d814641.png">

Hovering the bottom selected row:
<img width="1846" alt="Screen Shot 2022-07-05 at 6 32 25 PM" src="https://user-images.githubusercontent.com/35148959/177433591-fe770d18-bb60-42ca-bad2-1f027b93465a.png">

For this little gap, I'm thinking we might need some input from design. We're probably going to need to make a decision about how far the blue border spans for both selected and expanded rows, and for when you have adjacent rows that are expanded (not selected) and selected.

Reply:
> this will need to also have :focus

good catch. adding `:focus` to that selector works pretty well

Reply:
about the gap with the selected row bottom-border; my intention was to have the top border server as a consistent cue for the user for the _selected_ state.

i can re-add it to the last item, and try to get a feel about how it impact the experience.

for sure we're gonna need some help from designers :)